features/305-microservice/microservice.md

```diff
@@ -0,0 +1,85 @@
+# Microservice
+
+## Overview
+
+The document specifies the details of `type: http-service` of a deployment item.
+
+## Configuration of the deployment item
+
+```yaml
+deployments:
+  my-awesome-service:
+    type: http-service
+    path: /api/
+    port: 8888
+    context: ./backend/service
```

Comment:
Is the word `context` commonly used for docker? It seems a bit untrival to me. Would other name such as `docker-context` or `docker-build-path` more obvious?

Reply:
Yes. See https://docs.docker.com/compose/compose-file/#context